### PR TITLE
chore: add search

### PR DESCRIPTION
**Pull request by zregvart**
Work in progress, first pass at the Algolia search implementation. Very work in progress with very rough design at this point.

### CODE REVIEW

File: antora-ui-camel/src/partials/footer-scripts.hbs [hunk 1]
```diff
@@ -1,5 +1,6 @@
 <script src="{{uiRootPath}}{{asset 'js/site.js'}}"></script>
 <script async src="{{uiRootPath}}{{asset 'js/vendor/highlight.js'}}"></script>
 <script async src="{{uiRootPath}}{{asset 'js/vendor/svg4everybody.js'}}"></script>
+<script async src="https://cdn.jsdelivr.net/algoliasearch/3/algoliasearchLite.min.js"></script>
```
Comment (zregvart) [inline antora-ui-camel/src/partials/footer-scripts.hbs]: I think I'd like to serve this from the same server, i.e. not rely on cdn.jsdelivr.net and have cache busting like we have for other resources.

**Comment by zregvart** [comment]
@Nayananga this is very much work in progress, it illustrates the direction in for the search functionality I wish to go.

**Comment by zregvart** [comment]
Algolia recently released [v4 of the client](https://github.com/algolia/algoliasearch-client-javascript), it's much lighter than v3, we should look into using it here.

**Comment by Nayananga** [comment]
Hello Mr Zoran, may I take a look over it. I couldnt complete this task
back on gsoc2019.

Thanks.

**Comment by zregvart** [comment]
@Nayananga awesome :+1: this is just the UI for the search, we still need to combine this with the work you did in #74 to build the index. And this is just a start of the UI.

**Comment by zregvart** [comment]
Work continued on #163.

**Pull request closed without merging**